src/main/java/seedu/address/model/activity/Amount.java

```diff
@@ -27,7 +28,7 @@ public Amount(double amount) {
      */
     public static boolean isValidAmount(double test) {
         double d = Math.round(test * 100) / 100.0;
-        return d > 0 && d <= 1000000;
+        return d >= 0 && d <= 1000000;
     }
```

Comment:
I think we should do this instead to clean up all the inconsistencies in acceptable amounts properly:
```
public Amount(double amount) {
    checkArgument(isValidAmount(amount), MESSAGE_POSITIVE_ONLY);
    value = amount;
}
public static boolean isValidAmount(double test) {
    return test >= 0;
}
public static boolean isValidExpenseAmount(double test) {
    double d = Math.round(test * 100) / 100.0;
    return d > 0 && d <= 1000000;
}
```


1. `isValidAmount` would represent acceptable object values and become in line with the corresponding `isValidXXX` methods in all the other field classes (`Name`, `Phone` etc.).
2. `isValidExpenseAmount` would be used at the parser stage to process valid expense amounts. 
3. Might want to have a `isValidSettlementAmount` that has slightly different conditions used for settlement parsing if you want different constraints on acceptable settlement amounts

Admittedly not the cleanest OOP wise, but I think it's decent enough for the final deadline since we are unlikely to have any more new 'types' of expenses/amounts